### PR TITLE
Jfsg random fixes

### PR DESCRIPTION
**Pull request by jfsgomes**
- Removing "board" section from index page
- Updating footer in team page
- Changing Linkedin logo in "Board & Officers" section in index

### CODE REVIEW

**Comment by jfsgomes** [comment]
Looks like I messed up and branched from jfsg-nav-logo instead of the master. This shouldn't be a problem right?

**Comment by jfsgomes** [comment]
@frmendes don't forget this one

**Comment by pm-araujo** [comment]
:+1:

File: app/team.html [hunk 1]
```diff
@@ -228,17 +228,55 @@
       </div>
     </section>
 
-    <section class="Footer">
+  <section class="Footer">
```
Comment (frm) [inline app/team.html:231]: Is the indentation correct here?

**Pull request merged**